Unaligned 64-bit atomic operation on 32-bit arm machine

Trying to run docker image on Raspberry Pi 2 Model B (which has 32-bit CPU). It fails on registration attempt with following stack trace:
```
ezbookkeeping | 2023-04-20 14:10:49 [ERROR] System Error! because unaligned 64-bit atomic operation, r=b2e63496-63b4-89fe-0000-0004c0a800c7
ezbookkeeping | runtime/internal/atomic/unaligned.go:8 (0x1151b)
ezbookkeeping | runtime/internal/atomic/atomic_arm.s:258 (0x118a7)
ezbookkeeping | github.com/mayswind/ezbookkeeping/pkg/uuid/internal_generator.go:74 (0x6d86d7)
ezbookkeeping | github.com/mayswind/ezbookkeeping/pkg/uuid/internal_generator.go:53 (0x6d8583)
ezbookkeeping | github.com/mayswind/ezbookkeeping/pkg/uuid/uuid_container.go:32 (0x70b68f)
ezbookkeeping | github.com/mayswind/ezbookkeeping/pkg/services/base.go:46 (0x70b674)
ezbookkeeping | github.com/mayswind/ezbookkeeping/pkg/services/users.go:137 (0x70b668)
ezbookkeeping | github.com/mayswind/ezbookkeeping/pkg/api/users.go:65 (0x739e0b)
ezbookkeeping | github.com/mayswind/ezbookkeeping/cmd/webserver.go:265 (0x7469fb)
ezbookkeeping | github.com/gin-gonic/gin@v1.9.0/context.go:174 (0x73e383)
ezbookkeeping | github.com/mayswind/ezbookkeeping/pkg/middlewares/request_log.go:16 (0x73e360)
ezbookkeeping | github.com/mayswind/ezbookkeeping/cmd/webserver.go:258 (0x74690b)
ezbookkeeping | github.com/gin-gonic/gin@v1.9.0/context.go:174 (0x73e247)
ezbookkeeping | github.com/mayswind/ezbookkeeping/pkg/middlewares/request_id.go:26 (0x73e21c)
ezbookkeeping | github.com/mayswind/ezbookkeeping/cmd/webserver.go:258 (0x74690b)
ezbookkeeping | github.com/gin-gonic/gin@v1.9.0/context.go:174 (0x7117c3)
ezbookkeeping | github.com/gin-contrib/gzip@v0.0.6/handler.go:60 (0x71179c)
ezbookkeeping | github.com/gin-gonic/gin@v1.9.0/context.go:174 (0x73d8b7)
ezbookkeeping | github.com/mayswind/ezbookkeeping/pkg/middlewares/recovery.go:33 (0x73d890)
ezbookkeeping | github.com/mayswind/ezbookkeeping/cmd/webserver.go:258 (0x74690b)
ezbookkeeping | github.com/gin-gonic/gin@v1.9.0/context.go:174 (0x6ae8af)
ezbookkeeping | github.com/gin-gonic/gin@v1.9.0/gin.go:620 (0x6ae5bc)
ezbookkeeping | github.com/gin-gonic/gin@v1.9.0/gin.go:576 (0x6ae17f)
ezbookkeeping | net/http/server.go:2936 (0x4a36f7)
ezbookkeeping | net/http/server.go:1995 (0x49ee7f)
ezbookkeeping | runtime/asm_arm.s:831 (0x80c37)
```

--- a/pkg/uuid/internal_generator.go
+++ b/pkg/uuid/internal_generator.go
@@ -35,8 +35,8 @@ type InternalUuidInfo struct {
 
 // InternalUuidGenerator represents internal bundled uuid generator
 type InternalUuidGenerator struct {
-	uuidServerId   uint8
 	uuidSeqNumbers [1 << internalUuidTypeBits]uint64
+	uuidServerId   uint8
 }
 
 // NewInternalUuidGenerator returns a new internal uuid generator

